src/payment/ApplePay.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { injectIntl, intlShape } from '@edx/frontend-i18n';
+
+import { basketSelector } from './data/selectors';
+import { addMessage, MESSAGE_TYPES } from '../feedback';
+import { ApplePayButton, redirectToReceipt } from './apple-pay';
+import messages from './ApplePay.messages';
+
+function ApplePay(props) {
```

Comment:
Should this component be located under the `apple-pay` directory?

Reply:
It could be – or could be folded directly into ApplePayButton. This is basically the component - container split. I'll move in the directory and rename it ApplePayButtonContainer or something. wdyt?

Reply:
Yeah, I think I like that. I'm only bringing this up because I think I'll go with a similar layout for the PayPal button which I'm implementing now.